v0/destinations/attribution/transform.js

```
@@ -0,0 +1,85 @@
+const get = require("get-value");
+const { destinationConfigKeys, batchEndpoint } = require("./config");
+const {
+  defaultPostRequestConfig,
+  defaultRequestConfig,
+  removeUndefinedAndNullValues,
+  getFieldValueFromMessage
+} = require("../../util");
+
+function responseBuilderSimple(payload, attributionConfig) {
+  const basicAuth = Buffer.from(`${attributionConfig.writeKey}:`).toString(
+    "base64"
+  );
+
+  const response = defaultRequestConfig();
+  const header = {
+    "Content-Type": "application/json",
+    Authorization: `Basic ${basicAuth}`
+  };
+  response.method = defaultPostRequestConfig.requestMethod;
+  response.headers = header;
+  response.body.JSON = removeUndefinedAndNullValues(payload);
+  response.endpoint = batchEndpoint;
+  response.userId = attributionConfig.userId;
+  response.statusCode = 200;
+
+  return response;
+}
+
+function getTransformedJSON(message, attributionConfig) {
+  const { type, anonymousId } = message;
```

We should throw error if type is not present.